tidy3d/components/spice/analysis/dc.py

```diff
@@ -65,3 +73,10 @@ class IsothermalSteadyChargeDCAnalysis(Tidy3dBaseModel):
         "at `convergence_dv` intervals until the required bias is reached. This is, therefore, a "
         "convergence parameter in DC computations.",
     )
+
+    fermi_dirac: bool = pd.Field(
+        False,
+        title="Fermi-Dirac statistics",
+        description="Determines whether Fermi-Dirac statistics are used. When False, "
+        "Boltzmann statistics will be used.",
```

Comment:
Maybe mention here and in changelog under what conditions users should switch to Fermi-Dirac statistics?